exercises/isogram/.meta/solutions/isogram.rb

```diff
@@ -1,9 +1,9 @@
 module BookKeeping
-  VERSION = 3
+  VERSION = 4
 end
 
 class Isogram
-  def self.is_isogram?(str)
+  def self.isogram?(str)
```

Comment:
Do use "string" as a name.   Better use phrase or word or whatever the most general "language" thing is that is wanted, rather than a programming thing.

Yet, even without that change, the example is not shown actively, so it would be acceptable to the "the problem is solvable in this language" goal.